content/W2016/W16-event-code-party.md

```diff
@@ -0,0 +1,21 @@
+title: Code Party
+Date: 2016-04-02
+Category: Events
+Tags: code party, social, open source, academic
+Slug: code-party-w16
+Author: Anna Lorimer
+Summary: A casual day of fun and work!
+
+Come out for a fun day of coding with fellow women in CS! Work on that project
+you've been wanting to work on, keep grinding on the assignment that's due soon,
+study with friends, or hang out and socialize with people!
+
+We'll be providing lunch, snacks (vegetarian, vegan and gluten-free options will be available) and some ideas of things you can work on. All levels of
+programmers welcome!
+
+## Event Details ##
+
++ **Who:** [Cis & trans women and non-binary individuals](http://wics.uwaterloo.ca/faq)
++ **What:** Code Party
++ **Where:** STC 0050
++ **When:** Sat. April 2, 10:00AM&ndash;5:00PM
```

Comment:
Could you make the date like the following?
Saturday April 2, 10:00AM&ndash;5:00PM

Just to keep it consistent 😄